Makefile

```diff
@@ -121,6 +122,18 @@ stop-dev:
 	@echo "### [Stopping dev environment] ###"
 	@$(CONTAINER_ENGINE)-compose -f $(DEPLOYMENTS_DIR)/docker-compose/docker-compose.yaml down
 
+
+# Tests
+.PHONY: test
+test:
+	@[[ -d $(TEST_DIR) ]] || mkdir $(TEST_DIR)
+	@go test ./... -coverprofile $(TEST_DIR)/cover.out
```

Comment:
It would be nice to have a `-race` option
https://go.dev/doc/articles/race_detector#Usage
WDYT?

Reply:
By the moment I think we are not using GOROUTINES, but I like the idea to include the flag to enhance the tests and cover the data race errors when we improve the components.

Reply:
Changed commited